py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -208,3 +209,25 @@ def mean(
     )
     set_layer_name(layer, target, name, source_ir)
     return layer.get_output(0)
+
+
+def any(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+    dim: Sequence[int] = [],
+    keepdim: bool = False,
+) -> TRTTensor:
+    num_out = cast_trt_tensor(ctx, input_val, trt.float32, f"{name}_cast")
+    abs_out = impl.unary.abs(
+        ctx,
+        target,
+        source_ir,
+        f"{name}_abs",
+        num_out,
+    )
+    max_out = amax(ctx, target, source_ir, f"{name}_amax", abs_out, dim, keepdim)
```

Comment:
Consider using the Int32-Cast, Sum-Reduce approach to calculate the `any` operator:
https://github.com/pytorch/TensorRT/blob/16c031349c6a1af5a8408a817f2ef8542aa6f176/core/conversion/converters/impl/reduce.cpp#L12-L40

Reply:
Hey @gs-olive, regarding to this, what if we have an input tensor as [-1, 1], then reduce sum would be 0 and the result would be `false`. However, it should return `true` right? 
Thanks!

Reply:
Hi @bowang007 - thanks for the catch - this is a great point. I agree - the absolute value is helpful in light of that, and `Max` + `Sum` should have the same ultimate effect for this converter. I think the Int32 cast is likely still needed, however, since floating point discrepancies could affect the value of `0` when casting from Float to Bool.